sql/indexes.sql

```diff
@@ -403,6 +407,7 @@ create index IX_3078CBE6 on ResourcePermission (roleId, ctCollectionId);
 create index IX_F3870DDF on ResourcePermission (scope, ctCollectionId);
 
 create unique index IX_C0F6BCAC on Role_ (companyId, classNameId, classPK, ctCollectionId);
+create unique index IX_D303FC25 on Role_ (companyId, classNameId, classPK, type_, ctCollectionId);
```

Comment:
I wonder why the regen is creating indexes on the Group and Role tables... @brianchandotcom is there an outstanding run of `ant build-service-portal` that still needs to happen? This may be the cause of our CI failures.

Reply:
@pei-jung I might try making a new branch, getting rid of all your regen commits, and trying a new regen. I think CI is complaining that when it runs `build-service-portal`, these indexes are being removed.

Reply:
I went ahead and tried it out - now only the Address indexes are being generated: https://github.com/pei-jung/liferay-portal/pull/714/commits/1c594e48ec0dc1d38942d21ee916aa5ff2b53eee#diff-363d5c2ce1c0519c37de0d4d549ee80faa2b4f11c38c0f3f418d4cd76c773fb4